src/optim/FishLeg/fishleg.py

```diff
@@ -97,6 +97,8 @@ def __init__(
         damping: float = 5e-1,
         update_aux_every: int = 10,
         warmup_steps: int = 0,
+        method: str = "rank-1",
+        eps: float = 1e-4,
```

Comment:
not a big fan of this... maybe we could have a dictionary for method_parameters and inside the method we check if it contains the necessary parameters? ... not sure.. I dont like that this eps is only used for the antithetic